### PR TITLE
App type var

### LINKED ISSUE #21: App type resource is missing for the module
Below things needs to be done

1. Add an app_type variable for the module, with default value as empty string
2. If app_type var is empty then use the namespace value
3. Create a volterra app-type object in shared namespace
4. Add labels of ves.io/app_type to http loadbalancer object and also to all vk8s service object

### PR DESCRIPTION
**Pull request by cohenaj194**
closes https://github.com/volterraedge/terraform-volterra-app-delivery-network/issues/21

### CODE REVIEW

File: loadbalancer.tf [hunk 1]
```diff
@@ -40,12 +40,20 @@ resource "volterra_waf" "this" {
   }
 }
 
+resource "volterra_app_type" "this" {
+  name      = var.app_type != "" ? var.app_type : local.namespace
+  namespace = "shared"
+}
```
Comment (cohenaj194) [inline loadbalancer.tf:62]: Do we need to add any `Application Type Features` to this app type?

File: loadbalancer.tf [hunk 2]
```diff
@@ -40,12 +40,20 @@ resource "volterra_waf" "this" {
...
+
 resource "volterra_http_loadbalancer" "this" {
   name                            = format("%s-lb", var.adn_name)
   namespace                       = local.namespace
   description                     = format("HTTPS loadbalancer object for %s origin server", var.adn_name)
   domains                         = [var.app_domain]
   advertise_on_public_default_vip = true
+  annotations {
+    "ves.io/app_type" = var.app_type != "" ? var.app_type : local.namespaces
+  }
```
Comment (cohenaj194) [inline loadbalancer.tf:72]: `annotations` seemed to be the right choice here.  @madhukar32 should this be added under the `annotations` block or `labels` block?

**Review by madhukar32 — CHANGES_REQUESTED** [review]
Can you please run terraform-docs, to update the Input variable in Readme

**Comment by madhukar32** [comment]
For terraform-docs, you can probably run this command

```
terraform-docs md tbl . >> README.md
```

File: loadbalancer.tf (hunk 2 above)
Comment (cohenaj194) [reply loadbalancer.tf:72]: @madhukar32 said we should use labels

**Review by cohenaj194 — COMMENTED** [review]
verified working in AC

now we just  need to fix the doc checks and this is gtg

File: loadbalancer.tf (hunk 1 above)
Comment (cohenaj194) [reply loadbalancer.tf:62]: yes and added

**Pull request closed without merging**